util/scripts/lsfcsm/README.md

```diff
@@ -0,0 +1,52 @@
+### Integration with IBM Platform LSF with Cluster System Manager (CSM)
+
+To enable job-level integration of UnifyCR, where the UnifyCR daemon
+is automatically launched on each node during job startup and
+terminate once the job completes, additions must be made
+to the CSM prologue and epilogue Python scripts. Typically,
+these CSM scripts are found somewhere within the `/opt/ibm/csm`
+directory and named `privileged_prolog` and `privileged_epilog`.
+Consult with your LSF system administrator for more precise location
+information, and to make the changes described below.
+
+We provide helper scripts that launch and terminate a UnifyCR daemon
+(i.e., `unifycrd`) on a node. The scripts are:
+* launch: `<INSTALL_PREFIX>/sbin/unifycr_lsfcsm_prolog`
+* terminate: `<INSTALL_PREFIX>/sbin/unifycr_lsfcsm_epilog`
+
+Integration of the helper scripts requires editing the CSM prologue and
+epilogue scripts to support a new "unifycr" job allocation flag.
+Users may specify the allocation flag as an option to the `bsub`
+command:
+```shell
+[prompt]$ bsub -alloc_flags unifycr <other options> jobscript.lsf
```

Comment:
```suggestion
[prompt]$ bsub -alloc_flags "unifycr <other options>" jobscript.lsf
```

If I remember correctly multiple alloc flags need to be specified in quotes.

Reply:
The `<other options>` are other bsub options, not additional allocation flags. I'll make that more clear.

Reply:
I have jumped through all the hoops and get the final nod from Tundra gatekeeper. I expect to have full access to the machine and getting this started later this week, hopefully.